### PR TITLE
added makefile

### PR DESCRIPTION
**Pull request by meghanacosmos**
A basic one is added, let me what else I can add / changes that can be made.

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+filesIsPointer =  getAddresses.cpp identifier.cpp main.cpp
+filesScanProcesses = Scan.cpp main.cpp
+filesGetBaseAddresses = MemoryScan.cpp main.cpp
+filesRead = readAndGetContent.cpp main.cpp
+filesGetContent = readAndGetContent.cpp main.cpp
+rebulildables = main 
+ALL = getAddresses.cpp identifier.cpp Scan.cpp MemoryScan.cpp readAndGetContent.cpp main.cpp
```
Comment (shashwatj07) [inline makefile]: This needs to be updated every time we add a new file, right?
Can we not change this to ```*.cpp```?

File: makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+scanProcesses: $(filesScanProcesses)
+	@g++ $(filesScanProcesses) -o main && ./main
+getBaseAddress: $(filesGetBaseAddresses)
+	@g++ $(filesGetBaseAddresses) -o main && ./main
+read: $(filesRead)
+	@g++ $(filesRead) -o main && ./main 
+getContent: $(filesGetContent)
+	@g++ $(filesGetContent) -o main && ./main
+clean: $(rebulildables)
+	@rm $(rebulildables)
```
Comment (shashwatj07) [inline makefile]: Git wants a new line after this. It's a coding convention. Please add.

File: makefile [hunk 3]
```diff
@@ -0,0 +1,22 @@
+filesIsPointer =  getAddresses.cpp identifier.cpp main.cpp
+filesScanProcesses = Scan.cpp main.cpp
+filesGetBaseAddresses = MemoryScan.cpp main.cpp
+filesRead = readAndGetContent.cpp main.cpp
+filesGetContent = readAndGetContent.cpp main.cpp
```
Comment (shashwatj07) [inline makefile:5]: I'm assuming these are fine. Didn't bother checking. ;)

**Review by shashwatj07 — CHANGES_REQUESTED** [review]
Some points to look at.

File: makefile (hunk 1 above)
Comment (meghanacosmos) [reply makefile]: You mean for `ALL`, right?  Sounds good!

Comment (shashwatj07) [reply makefile]: Yes

**Comment by shashwatj07** [comment]
ping! @Meghana-12

**Comment by shashwatj07** [comment]
please rebase and resolve conflicts @Meghana-12

**Pull request closed without merging**